### ISSUE TITLE
Move vinn, py_vulcanize & py_trace_event to catapult/tools/

### ISSUE THREAD

**Issue by nedn** [catapult-project/catapult#1913]
@natduca @anniesullie

**Comment by natduca** [comment]
Fine by me.


Maybe we need a .md in the toplevel that explains the rules for these folders? experimental, catapult_base, third_party, and tools have emerging rules but its not written down anywhere

**Comment by anniesullie** [comment]
+1

@Apeliotes had you written any guidelines before?

**Comment by chromy** [comment]
As mentioned on an internal thread we would like to move `py_vulcanize` out of `third_party` to make it easier to build tracing when `third_party` is not available. Given the above I hope that's okay, I'm happy to do the work but I'm not sure where to move the directory, I can create `tools/` as mentioned in the title but maybe one of the existing directories is better? 

I guess the options are: 
- tools/py_vulcanize
- catapult_base/py_vulcanize
- catapult_build/py_vulcanize
- common/py_vulcanize

+@skyostil, @vinterstum

**Comment by nedn** [comment]
I vote for either `common/` or `tracing_build/`.

**Comment by anniesullie** [comment]
I prefer common because we may use py_vulcanize for perf dashboard in the future too.

**Comment by nedn** [comment]
That sgtm. Though Annie, should perf dashboard be using standard vulcanization tool instead of our home grown tool? 
IMO, it would be great if we can just kill py_vulcanize in the future & use whatever is the standard out there.

**Comment by anniesullie** [comment]
Yeah, the best thing to do would be to use [polymer-bundler](https://github.com/Polymer/polymer-bundler) to vulcanize everything. That way we could always use paper elements, etc. I haven't had time to investigate how difficult this would be to do.

**Comment by chromy** [comment]
Thanks! 😃 Re polymer-bundler: that would be good (especially given issues like #3717). For now I'll try moving `py_vulcanize` to `common` to see how hard it is.

**Comment by vinterstum** [comment]
chromy@: I landed your patch (with a tiny fix) while you were OOO; thanks :)
